app/models/trends/query.rb

```diff
@@ -1,15 +1,13 @@
 # frozen_string_literal: true
 
 class Trends::Query
-  include Redisable
   include Enumerable
 
-  attr_reader :prefix, :klass, :loaded
+  attr_reader :klass, :loaded
 
   alias loaded? loaded
 
-  def initialize(prefix, klass)
-    @prefix  = prefix
+  def initialize(_prefix, klass)
```

Comment:
If `_prefix` is no longer used, why not remove it altogether?

Reply:
The whole `Trends::Query` class is a candidate for removal since now it's just a wrapper around ActiveRecord now (it used to wrap Redis access), but maybe this could be the first step.